tools/cli/templates/projects/webpack/webpack.config.js

```diff
@@ -0,0 +1,64 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const path = require('path');
+
+/* eslint-disable max-lines-per-function */
+module.exports = (env, argv) => {
+    const isProd = env.production === true;
+
+    const config = {
+        stats: {
+            children: true,
+        },
+        entry: path.resolve(__dirname, './index.js'),
+        output: {
+            path: path.resolve(__dirname, './dist'),
+            filename: 'index_bundle.js',
+        },
+        mode: 'development',
+        plugins: [new HtmlWebpackPlugin({
+            template: path.resolve(__dirname, './index.html'),
+        }),
+        new MiniCssExtractPlugin()],
+        module: {
+            rules: [
+                {
+                    test: /\.(?:js|mjs|cjs)$/,
+                    exclude: /node_modules/,
+                    use: {
+                        loader: 'babel-loader',
+                        options: {
+                            presets: [
+                                ['@babel/preset-env', { targets: 'defaults' }],

```

Comment:
Do we need babel here? Does anything in the components JS needs to be transpiled?

Reply:
we don't need it - removed it